app/controllers/crud_controller.rb

```diff
@@ -1,4 +1,58 @@
-class CrudController < ApplicationController
-  def index
+  class CrudController < ApplicationController
```

Comment:
They should be more specific with the name of their controllers, for example I see that this controller is for employees, so I would recommend instead of calling it CrudController it be EmployeeController